src/components/Login.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import { View, Text, Image } from 'react-native';
+import firebase from 'firebase';
+import { Actions } from 'react-native-router-flux';
+import { Button, Input, CardSection, Spinner } from './common';
+import Logo from '../img/knapsack_logo.png';
+
+
+class Login extends Component {
+  state = {
+     email: '',
+     password: '',
+     error: '',
+     loading: false
+  }
+
+  onLogginSuccess() {
+    this.setState({ email: '', password: '', loading: false, error: '' });
+    Actions.main();
+  }
+
+  onLogginFail() {
+    this.setState({ error: 'Authentication Failed.', loading: false });
+  }
+
+
+  onButtonPress() {
+    const { email, password } = this.state;
+
+   this.setState({ error: '', loading: true });
+
+    firebase.auth().signInWithEmailAndPassword(email, password)
+      .then(this.onLogginSuccess.bind(this))
+      .catch(this.onLogginFail.bind(this));
+  }
+
+  renderLoginButton() {
+    if (this.state.loading) {
+      return <Spinner size="small" />;
+    }
+
+    return (
+      <Button
+        accessibility
+        Label={'Click to log in!'}
```

Comment:
`accessibility` and `Label` should be on the same line.

[Source](https://facebook.github.io/react-native/docs/button.html#accessibilitylabel)

Reply:
fixed